README.md

```diff
@@ -51,7 +51,7 @@ Installation
 
 You can download libfuse from
 https://github.com/libfuse/libfuse/releases. To build and install, we
-recommend to use [Meson](http://mesonbuild.com/) and
+recommend using [Meson](http://mesonbuild.com/) and
```

Comment:
Actually, this is outdated. Use of Meson and Ninja is required to today, not just recommended. Could you maybe update this as part of this pull request?

Reply:
Okay, I updated it to "must use".  (I haven't built libfuse myself, so I'm trusting you here :) )